doc/api/crypto.md

```diff
@@ -2102,12 +2120,16 @@ changes:
 -->
 
 * `type`: {string} Must be `'rsa'`, `'dsa'`, `'ec'`, `'ed25519'`, `'ed448'`,
-  `'x25519'`, or `'x448'`.
+  `'x25519'`, `'x448'`, or `'dh'`.
 * `options`: {Object}
   * `modulusLength`: {number} Key size in bits (RSA, DSA).
   * `publicExponent`: {number} Public exponent (RSA). **Default:** `0x10001`.
   * `divisorLength`: {number} Size of `q` in bits (DSA).
   * `namedCurve`: {string} Name of the curve to use (EC).
+  * `prime`: {Buffer} The prime parameter (DH).
+  * `primeLength`: {Buffer} Prime length in bits (DH).
```

Comment:
This really needs to be a Buffer? Isn't it a bit count, and it can't be larger than `8* prime.length` (where `prime` is the prev param)?

Reply:
Should have been `number`, thank you!